Adamant/Modules/Wallets/WalletViewControllerBase.swift

```diff
@@ -134,29 +135,13 @@ class WalletViewControllerBase: FormViewController, WalletViewController {
                 alert: self?.service?.core.wallet?.notifications,
                 isBalanceInitialized: self?.service?.core.wallet?.isBalanceInitialized ?? false
             )
-            
-            let height = $0.value?.fiat != nil ? BalanceTableViewCell.fullHeight : BalanceTableViewCell.compactHeight
-            
-            $0.cell.height = { height }
-        }.cellUpdate { [weak self] (cell, row) in
-            let symbol = self?.service?.core.tokenSymbol ?? ""
-            
-            row.value = self?.balanceRowValueFor(
-                balance: self?.service?.core.wallet?.balance ?? 0,
-                symbol: symbol,
-                alert: self?.service?.core.wallet?.notifications,
-                isBalanceInitialized: self?.service?.core.wallet?.isBalanceInitialized ?? false
-            )
-            
-            let height = row.value?.fiat != nil ? BalanceTableViewCell.fullHeight : BalanceTableViewCell.compactHeight
-            
-            cell.height = { height }
-            cell.titleLabel.text = BaseRows.balance.localized
```

Comment:
Why so much of code is removed?

Reply:
It was dead code, it never executes. Because line 158 overwrite cellUpdateCallback variable inside Eureka